pipelines/openjdk10_nightly_pipeline.groovy

```diff
@@ -0,0 +1,33 @@
+println "building OpenJDK10"
+stage('build OpenJDK') {
+    def Platforms = [:]
+    Platforms["Linux"] = {
+        build job: 'openjdk10_build_x86-64_linux'
+    }
+    Platforms["Mac"] = {
+        build job: 'openjdk10_build_x86-64_macos'
+    }
+    Platforms["Windows"] = {
+        build job: 'openjdk10_build_x86-64_windows'
+    }
+    Platforms["ppc64le"] = {
+        build job: 'openjdk10_build_ppc64le_linux'
+    }
+    Platforms["aix"] = {
+        build job: 'openjdk10_build_ppc64_aix'
+    }
+    Platforms["aarch64"] = {
+        build job: 'openjdk10_build_arm64_linux'
+    }
+    parallel Platforms
+}
+stage('checksums') {
+    node {
+        def job = build job: 'openjdk10_build_checksum'
+    }
```

Comment:
Not required to have any job wrapped in a node block. Any/all downstream jobs will get their own nodes.

Reply:
@AdamBrousseau Thanks for being patient with me :-).  I understand why we don't want to lock the master node when we're firing off downstream jobs that are running on other nodes. But in this case checksum and release tools are running on master.  Or is it the case that you're saying remove this anyway incase we shifted those jobs off master?

Reply:
Every job will grab its own node, whether it's master or any other node. Therefore you don't need to wrap any `build job:` step in  a node block.

Reply:
@karianna if you check the configure of job openjdk10_build_checksum you will see the job is restricted to node with master label https://ci.adoptopenjdk.net/job/openjdk10_build_checksum/configure 

![screen shot 2018-03-21 at 2 01 22 pm](https://user-images.githubusercontent.com/18030625/37728437-97e39a40-2d10-11e8-95c1-1e7a3ea26708.png)

Reply:
Right, so it's safe to remove the node{} block because the job will grab its own node (in this case master because that's what we've restricted it to).

Also as a side note I assume I can remove the `def job =` declaration as we don't reuse that var?

Reply:
Yes, for now we can remove the def job = .